wranglertools/get_field_info.py

```diff
@@ -396,8 +400,13 @@ def main():  # pragma: no cover
         sys.exit(1)
     connection = FDN_Connection(key)
     if args.type == ['all']:
-        args.type = [sheet for sheet in sheet_order if sheet not in [
-                    'ExperimentMic_Path', 'OntologyTerm']]
+        excluded_types = ['ExperimentMic_Path', 'OntologyTerm']
+        if not connection.admin:
+            excluded_types.extend([
+                'User', 'Lab', 'Award', 'Organism', 'FileFormat', 'FileReference',
```

Comment:
I'm not sure we really want FileReference to be in excluded_types if the user isn't admin - this is used for things like fastq barcodes now or fish probe sequences so may often be needed for experiment metadata submission.